posthog/api/test/__snapshots__/test_decide.ambr

```diff
@@ -589,7 +589,9 @@
 ---
 # name: TestDecide.test_flag_with_regular_cohorts.5
   '
-  SELECT ("posthog_person"."properties" -> '$some_prop_1') = '"something_1"' AS "flag_X_condition_0"
+  SELECT (("posthog_person"."properties" -> '$some_prop_1') = '"something_1"'
```

Comment:
Why did these change?

Reply:
I have no idea. I think someone commited something with bad snapshots

Reply:
So...can we merge this safely?

Reply:
yes